environments.json

```diff
@@ -4936,7 +4936,74 @@
     "test_suites": [
       "es5",
       "es6",
-      "es2016plus"
+      "es2016plus",
```

Comment:
You don't need to add the `test_suites` field if all the test suites are enabled, this is the default.